### PR TITLE
Quick start improvements

### PR DESCRIPTION
**Pull request by Reinmar**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: Quick start improvements.

---

### Additional information

I made commits one by one to make clear why I change those things.

There is one thing left to fix which is this paragraph:

> Copy these files to your project directory. You may use the \[CDN configuration\](#installing-ckeditor-5-from-cdn) as an example. You can also refer to framework integration guides for sample implementations.

I don't actually even understand what we're trying to say here.

I'd see it more less like this:

> Copy these files to your project directory (e.g. to a `vendor/` directory).
> 
> You can open the `index.html` file to check the sample included in the ZIP archive. Note, that you need to do that via an HTTP server.
> 
> Using the CKEditor&nbsp;5 via a ZIP archive is nearly identical to the other two installation methods (npm and CDN). You can use the \[CKEditor&nbsp;5 Builder\](https://ckeditor.com/ckeditor-5/builder) to create your custom editor setup and combine this with the editor loaded from files included in the ZIP archive.

It's still not as smooth as it could be. But we'll improve this once Builder will have the "ZIP file" category in the "Integration methods" picker.

### CODE REVIEW

File: docs/getting-started/quick-start.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ You have a few methods to choose from:
 * [Using CKEditor&nbsp;5 Builder](#using-ckeditor-5-builder) for the smoothest setup with live preview and multiple integration options.
 * [Using npm](#installing-ckeditor-5-using-npm) where you use a JavaScript package and build the editor with a bundler.
 * [Using CDN](#installing-ckeditor-5-from-cdn), where you use our cloud-distributed CDN in a no-build setup.
-* [Using a provided JavaScript file](#installing-ckeditor-5-from-a-file) where you download the ready-to-run file and copy them to your project.
+* [Using a provided JavaScript file](#installing-ckeditor-5-from-a-zip-file) where you download the ready-to-run file and copy them to your project.
```
Comment (Reinmar) [inline docs/getting-started/quick-start.md:19]: Maybe the title of this link should also mention ZIP?

> Using a ZIP file

There st of the description ("where you download...") describes the rest.

Comment (gorzelinski) [reply docs/getting-started/quick-start.md:19]: Yes, it should. I'm changing things currently, and it is in my changes.

File: docs/getting-started/quick-start.md [hunk 2]
```diff
@@ -192,10 +192,10 @@ Your final page should look similar to the one below.
 
 ## Installing CKEditor&nbsp;5 from a ZIP file
 
-If you do not want to build your project using npm, and you cannot rely on the CDN delivery, you can download ready-to-run files with CKEditor&nbsp;5 and all its plugins.
+If you do not want to build your project using npm and cannot rely on the CDN delivery, you can download ready-to-run files with CKEditor&nbsp;5 and all its plugins.
 
 1. <a href="https://cdn.ckeditor.com/ckeditor5/{@var ckeditor5-version}/zip/ckeditor5-{@var ckeditor5-version}.zip">Download the ZIP archive with the latest CKEditor&nbsp;5 distribution.</a>
-2. Extract the ZIP archive into a dedicated directory inside your project. It is recommended to include the editor version in the directory name to ensure proper cache invalidation once a new version of CKEditor&nbsp;5 is installed.
+2. Include the editor version in the directory name to ensure proper cache invalidation whenever you install a new version of CKEditor&nbsp;5.
```
Comment (Reinmar) [inline docs/getting-started/quick-start.md:198]: Why did you remove the "Extract ..." part?

Now the instruction says "download" and then "include". It seems incomplete.

**Comment by Reinmar** [comment]
> There is one thing left to fix which is this paragraph:
> 
> > Copy these files to your project directory. You may use the [CDN configuration](#installing-ckeditor-5-from-cdn) as an example. You can also refer to framework integration guides for sample implementations.

@gorzelinski, could you also apply this suggestion from the PR description?

File: docs/getting-started/quick-start.md [hunk 3]
```diff
@@ -208,7 +208,15 @@ Files included in the .zip archive:
...
-Copy these files to your project directory. You may use the [CDN configuration](#installing-ckeditor-5-from-cdn) as an example. You can also refer to framework integration guides for sample implementations.
+Copy necessary files to your project directory (e.g., to a `vendor/` directory).
+
+The easiest way to see the editor in action is to serve the `index.html` file via an HTTP server.
+
+<info-box warning>
+	You must run your code on a local server to use import maps. Opening the HTML file directly in your browser will trigger security rules. These rules (CORS policy) ensure loading modules from the same source. Therefore, set up a local server, like `nginx`, `caddy`, `http-server`, to serve your files over HTTP or HTTPS.
+</info-box>
+
+All three installation methods - npm, CDN, ZIP - work similarly. So, you can also use the [CKEditor&nbsp;5 Builder](https://ckeditor.com/ckeditor-5/builder/) with a ZIP archive. Create a custom preset with the Builder and combine it with the editor loaded from ZIP files.
```
Comment (Reinmar) [inline docs/getting-started/quick-start.md:219]:
```suggestion
All three installation methods &ndash; npm, CDN, ZIP &ndash; work similarly. So, you can also use the [CKEditor&nbsp;5 Builder](https://ckeditor.com/ckeditor-5/builder/) with a ZIP archive. Create a custom preset with the Builder and combine it with the editor loaded from ZIP files.
```

File: docs/getting-started/quick-start.md [hunk 4]
```diff
@@ -382,29 +390,37 @@ Your final page should look similar to the one below.
...
-Copy these files to your project directory. You may use the [CDN configuration](#installing-premium-features-from-cdn) as an example. You can also refer to framework integration guides for sample implementations.
+Copy necessary files to your project directory (e.g., to a `vendor/` directory).
+
+The easiest way to see the editor in action is to serve the `index.html` file via an HTTP server.
+
+<info-box warning>
+	You must run your code on a local server to use import maps. Opening the HTML file directly in your browser will trigger security rules. These rules (CORS policy) ensure loading modules from the same source. Therefore, set up a local server, like `nginx`, `caddy`, `http-server`, to serve your files over HTTP or HTTPS.
+</info-box>
+
+All three installation methods - npm, CDN, ZIP - work similarly. So, you can also use the [CKEditor&nbsp;5 Builder](https://ckeditor.com/ckeditor-5/builder/) with a ZIP archive. Create a custom preset with the Builder and combine it with the editor loaded from ZIP files.
```
Comment (Reinmar) [inline docs/getting-started/quick-start.md:423]:
```suggestion
All three installation methods &ndash; npm, CDN, ZIP &ndash; work similarly. So, you can also use the [CKEditor&nbsp;5 Builder](https://ckeditor.com/ckeditor-5/builder/) with a ZIP archive. Create a custom preset with the Builder and combine it with the editor loaded from ZIP files.
```

File: docs/getting-started/quick-start.md (hunk 2 above)
Comment (Reinmar) [reply docs/getting-started/quick-start.md:198]: I know there's "Copy necessary files" later on. But what's the point of "Include the editor version in the directory name" if we didn't ask them to extract the archive yet?

Also: asking them to copy **necessary** files creates a risk of copying to little, changing structure. KISS. \\

1.  Download
2.  Extract to `vendor/`. Make sure to have editor version yadadad
3.  Checkout the sample yadada

Comment (gorzelinski) [reply docs/getting-started/quick-start.md:198]: Good points - I'll change it.

**Comment by gorzelinski** [comment]
> BTW. I went for "ZIP archive" and "ZIP file". I found that version to be used often and e.g. on the download page we also have ZIP uppercased:
> 
> 
> 
> Also, I decided to use "archive" and "file" to have both covered just in case. IDK if it's gonna impact SEO, but better safe than sorry.

These are different keywords, so it won't harm to have both versions. Uppercased ZIP is ok. I used it in other ZIP guides. Now, it's unified. It also doesn't stand so much from the CDN heading in ToC.

**Review by gorzelinski — COMMENTED** [review]
I think I addressed all the comments. You can let me know if there is anything else I can help with.
